terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/webdrivers/ChromeDriverFactoryBean.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2013-2018 NTT DATA Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+package org.terasoluna.gfw.functionaltest.app.webdrivers;
+
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.springframework.beans.factory.FactoryBean;
+
+public class ChromeDriverFactoryBean implements FactoryBean<ChromeDriver> {
+
+    @Override
+    public ChromeDriver getObject() {
+        return new ChromeDriver();
+    }
+
+    @Override
+    public Class<?> getObjectType() {
+        return ChromeDriver.class;
+    }
+
+    @Override
+    public boolean isSingleton() {
+        return false;
+    }
+
+}
```

Comment:
- [ ] Unnecessary changes. (Unnecessary factory bean.)

Reply:
returned a fix for the added factory bean.
https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/pull/806/commits/af8a63b38e21e4f50b51fe8110901876c485f5b4